LanguageSpecification.md

```diff
@@ -759,10 +807,10 @@ Mapping one from set of numbers to another is available with the default being l
 const a = map b from [0 in, 100 in] to [0 %, 100%]
 ```
 
-This will extraplote linearlly in the range if an input outside the domain is provided. The following has identical behavior:
+This will extrapolate linearly in the range if an input outside the domain is provided. The following has identical behavior:
 
 ```
-const a = map b from [0 in, 100 in] to [0 %, 100%] linearlly
+const a = map b from [0 in, 100 in] to [0 %, 100%] linearly
 ```
 
 Additionally logarithmically and exponentially are available.
```

Comment:
Please check if sigmoid and quadratic available. I don’t think exponential is still there. Please see `MappingBuilder.java` and `guides/two_trees.html`.